src/tab.group.ts

```diff
@@ -77,7 +79,7 @@ export default class TabGroup extends LitElement {
               data-test="overflow-start-button"
               tabindex="-1"
               ?disabled=${this.isDisableOverflowStartButton}
-              @click=${this.#onOverflowStartButtonClick}
+              @click=${this.#onOverflowButtonClick.bind(this, 'start')}
```

Comment:
`#onOverflowEndButtonClick()` was just an indirection that called `#scrollTabsList()`. So I removed it as a middle-man and renamed `#scrollTabsList()` to `#onOverflowEndButtonClick()`.